### PR TITLE
test case implementation example for JavaScript Validators 

### PR DESCRIPTION
**Pull request by yusuke**
test case implementation example for JavaScript Validators.
redpen.js : utlity script for testing JavaScript validators
validator-js-test.js : tests validator.js

### CODE REVIEW

File: redpen-cli/sample/js/test/validator-js-test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ 3. rename validator.js.example to enable the validator implementation
+ $ cd $REDPEN_HOME/sample
+ $ mv validator.js.example validator.js
+
+ 4. run mocha
+ $ cd $REDPEN_HOME/sample
+ $ mocha
+
+ */
+assert = require('assert');
```
Comment (azu) [inline redpen-cli/sample/js/test/validator-js-test.js]: Why is `assert` variable global?

I think that `var assert = require('assert');` is better. 

- http://stackoverflow.com/questions/2485423/is-using-var-to-declare-variables-optional

File: redpen-cli/sample/js/test/validator-js-test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+redpen = require('./redpen');
```
Comment (azu) [inline redpen-cli/sample/js/test/validator-js-test.js]: ditto.

Comment (yusuke) [inline redpen-cli/sample/js/test/validator-js-test.js]: thanks!
fixed.

File: redpen-cli/sample/js/test/validator-js-test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+steps to run this test:
+ 1. install mocha
+ $ npm install -g mocha
+
+ 2. run RedPen in server mode
+ $ cd $REDPEN_HOME/bin
+ $ ./redpen-server
+
+ 3. rename validator.js.example to enable the validator implementation
+ $ cd $REDPEN_HOME/sample
```
Comment (takahi-i) [inline redpen-cli/sample/js/test/validator-js-test.js]: I think there is no **sample** directory in $REDPEN_HOME. **js** directory is correct.

File: redpen-cli/sample/js/test/validator-js-test.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ $ mv validator.js.example validator.js
+
+ 4. run mocha
+ $ cd $REDPEN_HOME/sample
```
Comment (takahi-i) [inline redpen-cli/sample/js/test/validator-js-test.js]: As the above example, **js** directory is correct.

**Comment by takahi-i** [comment]
Thank you a lot for the contribution @yusuke as always! I have verified the changes are fine. I just think the directory name does not exist in the REDPEN_HOME as I commented.

**Pull request merged**